plugins/woocommerce/client/legacy/js/frontend/wp-consent-api-integration.js

```diff
@@ -0,0 +1,24 @@
+( function ( $ ) {
+	'use strict';
+
+	// check init order source attribution on consent change
+	const consentCategory = 'marketing';
```

Comment:
Since this is used as an actual constant string that never changes I'd probably change it to something like this (changing its name where it's used as well): 

```suggestion
	const CONSENT_CATEGORY_MARKETING = 'marketing';
```

Reply:
✔️  9304b6e710508119b7a6e63dc66821063117d7c2